shared/middleware/animation_library_api.rb

```diff
@@ -19,6 +19,26 @@ class AnimationLibraryApi < Sinatra::Base
     load(CDO.dir('shared', 'middleware', 'helpers', 'core.rb'))
   end
 
+  #
+  # GET /api/v1/animation-library/levelbuilder/<version-id>/<filename>
```

Comment:
[i'm not sure how strongly i feel about this, so let me know your thoughts] i worry that the `levelbuilder` directory will be confused for the levelbuilder environment. in many of our S3 buckets, we have separate directories for environments, like in `cdo-v3-sources`:
<img width="247" alt="Screen Shot 2020-07-23 at 3 19 13 PM" src="https://user-images.githubusercontent.com/9812299/88344196-e9387080-ccf7-11ea-81c3-120ba6a86e95.png">

workshopping other possible names: custom, custom_levels, levels, internal (meh)

Reply:
Yeah I was thinking that a little bit too, but wasn't sure what the best name would be. Changed to `level_animations`, does that work?

Reply:
yeah, i like that!

Reply:
just noticed that this comment needs to be updated to `level_animations` as well

Reply:
whoops, thanks!